packages/hub/src/lib/commit.spec.ts

```diff
@@ -55,129 +127,77 @@ describe("commit", () => {
-				credentials: { accessToken: TEST_ACCESS_TOKEN },
-			});
-		}
-	}, 30_000);
+			],
+		});
 
-	it("should commit a full repo from HF with web urls", async function () {
-		const repoName = `${TEST_USER}/TEST-${insecureRandomString()}`;
-		const repo: RepoId = {
-			name: repoName,
-			type: "model",
-		};
+		const webResourceContent = await downloadFile({ repo, path: "lamaral.json" });
+		assert.strictEqual(webResourceContent?.status, 200);
+		assert.strictEqual(await webResourceContent?.text(), await (await fetch(tokenizerJsonUrl)).text());
+	}, 20_000);
 
-		await createRepo({
+	it("should commit a full repo from HF with web urls", async function () {
+		const FILES_TO_UPLOAD = [
+			`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/model.json`,
+			`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/group1-shard1of2`,
+			`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/group1-shard2of2`,
+			`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/coffee.jpg`,
+			`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/README.md`,
+		];
+
+		const operations: CommitFile[] = await Promise.all(
+			FILES_TO_UPLOAD.map(async (file) => {
+				return {
+					operation: "addOrUpdate",
+					path: file.slice(file.indexOf("main/") + "main/".length),
+					// upload remote file
+					content: new URL(file),
+				};
+			})
+		);
+		await commit({
+			repo,
 			credentials: {
 				accessToken: TEST_ACCESS_TOKEN,
 			},
-			repo,
+			title: "upload model",
+			operations,
 		});
 
-		try {
-			const FILES_TO_UPLOAD = [
-				`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/model.json`,
-				`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/group1-shard1of2`,
-				`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/group1-shard2of2`,
-				`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/coffee.jpg`,
-				`https://huggingface.co/spaces/huggingfacejs/push-model-from-web/resolve/main/mobilenet/README.md`,
-			];
-
-			const operations: CommitFile[] = await Promise.all(
-				FILES_TO_UPLOAD.map(async (file) => {
-					return {
-						operation: "addOrUpdate",
-						path: file.slice(file.indexOf("main/") + "main/".length),
-						// upload remote file
-						content: new URL(file),
-					};
-				})
-			);
-			await commit({
-				repo,
-				credentials: {
-					accessToken: TEST_ACCESS_TOKEN,
-				},
-				title: "upload model",
-				operations,
-			});
-
-			const LFSSize = (await fileDownloadInfo({ repo, path: "mobilenet/group1-shard1of2" }))?.size;
+		const LFSSize = (await fileDownloadInfo({ repo, path: "mobilenet/group1-shard1of2" }))?.size;
 
-			assert.strictEqual(LFSSize, 4_194_304);
-		} finally {
-			if (!Math.random()) {
```

Comment:
Why was it there ?

Reply:
I use it to write code that returns/throws without the linter being unhappy about unreachable code